accname/name/comp_label.html

```diff
@@ -18,7 +18,7 @@
 <!-- Todo: test all remaining cases of https://w3c.github.io/accname/#comp_label -->
 
 <script>
-AriaUtils.verifyLabelsBySelector(".ex");
+AriaUtils.verifyLabelsBySelector(1, ".ex");
```

Comment:
See below... Keep `comp_label.html` as-is (but with a warning) to make way for https://github.com/web-platform-tests/wpt/pull/41463

Reply:
@jugglinmike #41463 was also merged since the the time this was filed.